client/tslint.prod.json

```diff
@@ -1,4 +0,0 @@
-{
-  "extends": ["./tslint.json"],
-  "defaultSeverity": "error"
-}
```

Comment:
don't we need this for eslint, too?

Reply:
CRA handles this [here](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/config/webpack.config.js#L772). So it's the other way around: Warnings are also errors during development now, except when you set the env variable `ESLINT_NO_DEV_ERRORS`. We could make the later default with craco.

Reply:
okay, we should do this because it's really annoying when the compilation aborts because of warnings...